src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -1955,6 +1955,61 @@ public function get_test_rest_availability() {
 		return $result;
 	}
 
+	/**
+	 * Test if 'file_uploads' directive in PHP.ini is turned off
+	 *
+	 * @since 5.5.0
+	 *
+	 * @return array The test results.
+	 */
+	public function get_test_file_uploads() {
+		$result = array(
+			'label'       => __( 'File uploads is on in PHP ini' ),
```

Comment:
Let's reduce the technicality of this line `Files can be uploaded`.

Reply:
Great idea - I would have suggested "File uploads are enabled from PHP ini", but that is still too much technical information for a Site Health section.